eZ/Publish/Core/MVC/Symfony/View/RelationView.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\MVC\Symfony\View;
+
+/**
+ * A view that contains relation data.
+ */
+interface RelationView
```

Comment:
This part we considered to rather get users to do from templates / view manually right? As it's not always relevant to tag the relation. If so we would need to start using for instance [fos_httpcache_tag](http://foshttpcachebundle.readthedocs.io/en/latest/features/tagging.html#tagging-from-twig-templates).